jsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
is this config in use?

Reply:
Yes. VS Code reads it, so the absolute paths in imports can be resolved with a click.